release_notes.md

```diff
@@ -3,10 +3,11 @@
 <!-- Please add your release notes in the following format:
 - My change description (#PR)
 -->
+- Update Python Worker Version to [4.13.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.13.0)
 - Update Python Worker Version to [4.12.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.12.0)
```

Comment:
Just to double check, if we need to remove this 4.12.0?

Reply:
Nope, new versions just get appended to the list